AdyenTests/Card Tests/Utilities/CardEncryptorCardTests.swift

```diff
@@ -21,14 +25,13 @@ class CardEncryptorCardTests: XCTestCase {
         let card = CardEncryptor.Card(number: "test_number")
         let key = "test_invalid_key"
         XCTAssertThrowsError(try card.encryptedNumber(publicKey: key, date: Date())) { error in
-            XCTAssertTrue(error is CardEncryptor.Error, "Thrown Error is not CardEncryptor.Error")
-            XCTAssertEqual(error as! CardEncryptor.Error, CardEncryptor.Error.encryptionFailed, "Thrown Error is not CardEncryptor.Error.encryptionFailed")
-            XCTAssertEqual(error.localizedDescription, CardEncryptor.Error.encryptionFailed.errorDescription)
+            XCTAssertTrue(error is Cryptor.Error, "Thrown Error is not CardEncryptor.Error")
+            XCTAssertEqual(error as! Cryptor.Error, Cryptor.Error.rsaEncryptionError, "Thrown Error is not CardEncryptor.Error.encryptionFailed")
+            XCTAssertEqual(error.localizedDescription, Cryptor.Error.rsaEncryptionError.errorDescription)
```

Comment:
Error types are part of the public interface, will client code be broken by this change?

Reply:
It is possible if client code expect to catch a specific type of the error.
I don't think it is a common thing to do, but to be on a safe side I can wrap `Cryptor.Error` into `CardEncryptor.Error. encryptionFailed(error: Cryptor.Error)`

Reply:
why would you think its uncommon, many clients build their own Card forms and use this code to encrypt card details before sending to us, I don't see a reason why they would be uncommon.

Reply:
Done. `CardEncryptor.Error.encryptionFailed` is deprecated :D

Reply:
still the merchants expect to receive `CardEncryptor.Error`, if its deprecated but is not used at all, it is still breaking change.

if an encryption error happens and merchants expecting `CardEncryptor.Error`, then their code is broken.

Reply:
Ok. I'll hide `Cryptor.Error` behind `CardEncryptor.Error.encryptionFailed`